app/models/like.rb

```diff
@@ -1,8 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
-  after_create :update_like_counter
+  # after_create :update_like_counter
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧